docs/dev/plugins/cloudformation/transformation/supported-types.md

```diff
@@ -0,0 +1,119 @@
+# Transformation by TOSCA Type
+
+Currently, the CloudFormation plugin only supports certain Node- and RelationshipTypes defined in the [TOSCA Simple Profile](http://docs.oasis-open.org/tosca/TOSCA-Simple-Profile-YAML/v1.1/cos01/). The following sections shows which types are supported and how those are mapped to CloudFormation.
+
+## Supported NodeTypes
+
+The following table contains the NodeTypes currently supported by the CloudFormation plugin and how they're mapped to CloudFormation resources.
+
+| TOSCA NodeType | CloudFormation |
+| --- | --- |
+| Apache | Setup through CloudFormation Init on the EC2 host |
+| Compute | EC2 resource and corresponding SecurityGroup |
+| Database | Setup through CloudFormation Init on the EC2 host |
+| Dbms | Setup through CloudFormation Init on the EC2 host |
+| MysqlDatabase | RDS resource with a mysql engine |
```

Comment:
Maybe add that an additional `SecurityGroup` is also created. Forgot this in the original document.